### PR TITLE
MAINTAINERS: add @258204 to collaborators

### LINKED ISSUE #7522: Add collaborator: @258204
Hi, @258204! You seem to be enjoying contributing to the tldr-pages project.
You now have had five distinct pull requests merged (#6131, #6132, #6133, #6134, #6137, #6145, #6547, #7484)!
That qualifies you to become a collaborator in this repository, as explained in our [community roles documentation](https://github.com/tldr-pages/tldr/blob/main/COMMUNITY-ROLES.md).

As a collaborator, you will have commit access to the repository.
That means you can merge pull requests, label and close issues, and perform various other maintenance tasks that are needed here and there.
Of course, all of this is voluntary — you're welcome to contribute to the project in whatever ways suit your liking.

If you do decide to start performing maintenance tasks, though, we only ask you to get familiar with the [maintainer's guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md).

So, what do you say? Can we add you as a collaborator?

Either way, thanks for all your work so far!

### PR DESCRIPTION
**Pull request by 258204**
Closes #7522

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ If you are an owner of the organization, you can see an automated list
   [27 Aug 2021](https://github.com/tldr-pages/tldr/issues/6415) — present
 - **Reinhart Previano Koentjoro ([@reinhart1010](https://github.com/reinhart1010))**:
   [23 Nov 2021](https://github.com/tldr-pages/tldr/issues/7404) — present
+- **258204 ([@258204](https://github.com/258204))**:
+  [12 December 2021](https://github.com/tldr-pages/tldr/issues/7522) — present
```
Comment (navarroaxel) [inline MAINTAINERS.md:54]: Today is 10th :grin: 
```suggestion
  [10 December 2021](https://github.com/tldr-pages/tldr/issues/7522) — present
```

Comment (258204) [reply MAINTAINERS.md:54]: :rofl:

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks for being a part of the team 🎉

**Review by sbrl — APPROVED** [review]
Welcome onboard, @258204! :D

**Comment by 258204** [comment]
Thanks everyone :D

**Comment by CleanMachine1** [comment]
Welcome @258204 !

**Pull request merged**